### PR TITLE
loadout - glasses and price adjustments

### PR DESCRIPTION
**Pull request by Hatterhat**
## About The Pull Request
adds modern glasses, which use a different glasses sprite that lets you actually kinda see the eyes below the glasses? looks a bit nicer maybe.

also drops a bunch of random polychromics' loadout costs to 2.

also also drops the red cloak (the one that looks like the goliath hide cloak) down to 2 points because why not

## Why It's Good For The Game
glasses: fashion choices
polychromics and other random stuff's costs: why not

## Changelog
:cl:
tweak: Polychromic clothing pieces have had their loadout prices toyed with.
tweak: The red cloak in loadout that looks like a goliath hide cloak also had its price dropped a bit.
add: Modern prescription glasses! Loadout only.
/:cl:

### CODE REVIEW

**Comment by Hatterhat** [comment]
@deathride58 quieres

File: modular_citadel/code/modules/client/loadout/suit.dm [hunk 1]
```diff
@@ -18,7 +18,6 @@
 /datum/gear/suit/redhood
 	name = "Red cloak"
 	path = /obj/item/clothing/suit/hooded/cloak/david
-	cost = 3
```
Comment (Linzolle) [inline modular_citadel/code/modules/client/loadout/suit.dm:21]: undocumented change?

File: modular_citadel/code/modules/client/loadout/uniform.dm [hunk 2]
```diff
@@ -280,36 +280,34 @@
...
-	cost = 3
+	cost = 2
 	loadout_flags = LOADOUT_CAN_NAME | LOADOUT_CAN_DESCRIPTION | LOADOUT_CAN_COLOR_POLYCHROMIC
 	loadout_initial_colors = list("#353535", "#808080", "#808080")
 
 /datum/gear/uniform/shorts/poly/athletic
 	name = "Polychromic Athletic Shorts"
 	path = /obj/item/clothing/under/shorts/polychromic
-	cost = 2
-	loadout_flags = LOADOUT_CAN_NAME | LOADOUT_CAN_DESCRIPTION | LOADOUT_CAN_COLOR_POLYCHROMIC
```
Comment (Linzolle) [inline modular_citadel/code/modules/client/loadout/uniform.dm:312]: did you mean to remove these?

Comment (Hatterhat) [reply modular_citadel/code/modules/client/loadout/uniform.dm:312]: yeah the shorts inherit the loadout flags and cost from the parent type up top

File: modular_citadel/code/modules/client/loadout/suit.dm (hunk 1 above)
Comment (Hatterhat) [reply modular_citadel/code/modules/client/loadout/suit.dm:21]: whoops

**Pull request merged**